content/a/ppw-hardcore-friday-the-13th-tokio.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "PpW Hardcore Friday the 13th 東京"
+template = "article.html"
+weight = 0
+authors = ["M3n747"]
+[extra.gallery]
++++
+
+PpW Hardcore Friday the 13th 東京 (_Tokio_) is an upcoming show by [PpW Ewenement Wrestling](@/o/ppw.md), set to take place on 13.03.2026 in Tokio, in a venue called ["Tokyo Square in Itabashi"][are-wa-tokyo-square-desu]. The show will be a cooperation with [Big Japan Pro Wrestling][bjw] and other local promotions.
```

Comment:
inconsistent romanization of the city name, probably should be Tokyo everywhere because that's the canonical English ver. Or we go the full Hepburn-shiki with _Tōkyō_